src/pythainer/examples/installs/__init__.py

```diff
@@ -237,3 +238,78 @@ def opencv_lib_install_from_src(
 
     if cleanup:
         builder.run(f"rm -rf {contrib_repo_name}")
+
+
+def tensor_rt_lib_install_from_deb(
+    builder: DockerBuilder,
+    workdir: str = "/tmp",
+    os: str = "ubuntu2204",
+    tag: str = "10.7.0-cuda-12.6",
+):
+    """
+    Install TensorRT library from a .deb package into a Docker image.
+
+    Parameters:
+        builder (DockerBuilder): An instance of DockerBuilder for executing commands.
+        workdir (str): Working directory inside the container where the package will be downloaded.
+        os (str): Operating system identifier, used in forming the download URL.
+        tag (str): Version tag of the TensorRT library, also used in forming the download URL.
+    """
+    tensorrt_download_url = f"https://developer.nvidia.com/downloads/compute/machine-learning/tensorrt/10.7.0/local_repo/nv-tensorrt-local-repo-{os}-{tag}_1.0-1_amd64.deb"
```

Comment:
you probably want to split `tag` into `cuda_version` and `tensorrt_version` such that you can also replace the `10.7.0` in that url?

Reply:
you can then "recreate `tag`" from these 2.